testing.md

```diff
@@ -55,7 +55,7 @@ See also:
 	- https://github.com/d-led/cpp-testing-no-excuses
```

Comment:
Can you make a separate PR for the changes in "testing.md" here together with the changes to Catch2 in https://github.com/MattPD/cpplinks/pull/34/files?

Reply:
@MattPD I've cherry-picked the commit there.